.vscode/extensions.json

```diff
@@ -1,10 +1,20 @@
 {
   "recommendations": [
-    "davidanson.vscode-markdownlint",
-    "dbaeumer.vscode-eslint",
-    "editorconfig.editorconfig",
-    "esbenp.prettier-vscode",
-    "unifiedjs.vscode-mdx",
-    "yoavbls.pretty-ts-errors"
+    // Formating
+    "esbenp.prettier-vscode", // Prettier
+    "editorconfig.editorconfig", // EditorConfig
+
+    // Highlighting
+    "bradlc.vscode-tailwindcss", // Tailwind CSS
+    "unifiedjs.vscode-mdx", // MDX
+    "shardulm94.trailing-spaces", // Trailing spaces
+    "styled-components.vscode-styled-components", // styled()
+
+    // Lint
+    "dbaeumer.vscode-eslint", // ESLint
+    "yoavbls.pretty-ts-errors", // TypeScript
+    "stylelint.vscode-stylelint", // Stylelint
+    "davidanson.vscode-markdownlint", // markdownlint
+    "chrischinchilla.vale-vscode" // Vale
```

Comment:
I'm not entirely sure about Vale. It's useful for us, but the extension is not standalone (requires installing Vale separately), which could be worse DX for external contributors.